WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -1148,6 +1152,12 @@ def finalize(self):
     elif res['Value'] and not requestFlag:
       # A request was created while there were none before, change final minor status
       self.__report(minorStatus=JobMinorStatus.PENDING_REQUESTS)
+      requestFlag = True
+
+    if not self.failedFlag:
+      self.__report(status=JobStatus.DONE)
```

Comment:
Here you can set the job as `Done / Pending requests`, right (in case there are pending requests)

Reply:
These 4 lines are reason why I removed the line 1141.

Reply:
I understood that, but the result is that successful jobs with requests would end up `Done` and not `Completed`...
and as said above accounting would not be with the status `Done`